src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -202,6 +204,8 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
     public static Network parseNetwork(String network) throws ParseException {
         requireNonNull(network);
         String trimmedNetwork = network.trim();
+        Logger logger = LogsCenter.getLogger(ParserUtil.class);
```

Comment:
nit: Can make logger a static field of ParserUtil. Also, would be better if more useful info & context is logged other than just a value.